tests/python/gaia-ui-tests/gaiatest/tests/functional/camera/test_front_camera_take_photo_with_timer.py

```diff
@@ -0,0 +1,49 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.camera.app import Camera
+from gaiatest.apps.camera.app import ImagePreview
+
+
+class TestCamera(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        # Turn off geolocation prompt
+        self.apps.set_permission('Camera', 'geolocation', 'deny')
+
+    def test_capture_a_photo(self):
+        """https://moztrap.mozilla.org/manage/case/15800/"""
+        self.previous_number_of_pictures = len(self.data_layer.picture_files)
+
+        self.camera = Camera(self.marionette)
+        self.camera.launch()
+
+        # Open Camera Settings and turn on timer
```

Comment:
Usually, if you have to put a comment within a test, that's a sign that the code is not correctly written. The main reason in this case is:  Usually, the comment just rephrases what the following line does.
For example:
``` python
# Open the XYZ panel
my_app.open_xyz_panel()
```
Having this kind of comments creates duplication, and hence, doubles the maintenance we have to do. For instance, if one day I change the lines to the following and I forget to update the comment:
``` python
# Open the XYZ panel
my_app.open_abc_panel()
```
The comment will lie to the future readers, who'll be confused. ("Where is the truth? Which panel should it open?")

So, to prevent that, let's just rely on the code.